docs/content.go

```diff
@@ -556,7 +559,10 @@ Enterprise Performance Computing (EPC)`
       docker://user/image:tag
     
   shub: Pull an image from Singularity Hub to CWD
-      shub://user/image:tag`
+      shub://user/image:tag
+
+  oras: Pull an SIF image from a supporting OCI registry
+      oras://registry/namespace/repo:tag`
```

Comment:
Should `repo` be `image` instead?

Reply:
The nomenclature I've seen more is to call that a repo and then the image is the repo plus tag.

i.e. https://hub.docker.com/_/golang, the repo is `golang` and then you specify which image you want based on the tag